### DISCUSSION TITLE
在这里放一下我在8月1日制作的有关绝区零的8bit音乐（第三章——拯救大黑客芮恩）

### DISCUSSION THREAD

**Discussion by ljyawtgd** [share121/inter-knot#5370] [绝区零]
https://github.com/user-attachments/assets/369676ee-61ee-4566-a05f-c9a32a4050a8

**由于只能上传10mb以下的，音质不知压成什么鬼样了**
[https://www.bilibili.com/video/BV16QvvemEuW](url)
**_使用软件：Famistudio_**

**Comment by March-777th** [comment]
# 我就说绳网里有大佬吧

**Comment by ZionHuang28** [comment]
咱们绳匠真是太强了

**Comment by KELERSAMA** [comment]
大佬
